lib/app/features/competiton/searched_results/result_details/widgets/tabs/stats_tab.dart

```diff
@@ -0,0 +1,93 @@
+import 'package:flutter/material.dart';
+import 'package:score_live/app/common/extensions/context/applocalization_context.dart';
+import 'package:score_live/app/common/extensions/enums.dart';
+import 'package:score_live/presentation/constants/app_colors.dart';
+import 'package:score_live/presentation/constants/app_const_variables.dart';
+import 'package:score_live/presentation/constants/text_styles.dart';
+
+import '../../../../../../../models/topscorers_model/topscorers_model.dart';
+
+class StatsTab extends StatelessWidget {
+  const StatsTab({super.key, required this.topScorers, required this.statsSwitch});
+
+  final StatsSwitch statsSwitch;
+  final List<TopScorersModel> topScorers;
+
+  @override
+  Widget build(BuildContext context) {
+    
+    if (topScorers.isEmpty) {
+      return SizedBox(
+        height: 200,
+        child: Center(
+          child: Text(
+            context.localizations.noStats,
+            textAlign: TextAlign.center,
+            style: const CustomTextStyle(color: Colors.white, fontSize: 24, fontWeight: FontWeight.w700),
+          ),
+        ),
+      );
+    }
+    return Column(
+      children: [
+        const StatsHeaderWithSwitch(),
+        Builder(
+          builder: (context) => switch (statsSwitch) {
+            StatsSwitch.goals => Container(
+                padding: const EdgeInsets.all(12.0),
+                color: AppColors.listTileGrey,
+                child: const Column(
+                  children: [],
```

Comment:
Why is the `children`  list empty?

Reply:
there will be a display of Api data, I will add `TODO` there